<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InformacionImagen</class>
 <widget class="QDialog" name="InformacionImagen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>198</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Información de la imagen</string>
  </property>
  <widget class="QLabel" name="labelSize">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Tamaño de la imagen (ancho x alto): </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSizeValue">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelProfundidad">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Profundida del pixel: </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelProfundidadValue">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelCanales">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Número de canales: </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCanalesValue">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelMemoria">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Tamaño en memoria: </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMemoriaValue">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelRGBMedia">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Valores RGB medios: </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRGBMediaValue">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelColorMedio">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Representación del color medio:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelColorMedioValue">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
